packages/connect/src/data/connectSettings.ts

```diff
@@ -154,5 +154,7 @@ export const parseConnectSettings = (input: Partial<ConnectSettings> = {}) => {
         settings.binFilesBaseUrl = input.binFilesBaseUrl;
     }
 
+    settings.enableFirmwareHashCheck = Boolean(input.enableFirmwareHashCheck);
```

Comment:
don't we want to keep the same structure as with other fields?

Reply:
No problem. My idea was, that if we have an optional param of `boolean` type specifically ,
we don't need `true | false | undefined`, and can instead default `undefined` to `false`, as they have the same meaning.
It was a suggestion, but for the sake of consistency I will use the old pattern.

:thought_balloon:  though if you agree with the proposed pattern, I can do that in a separate PR for _all_ optional booleans :slightly_smiling_face:

Reply:
arrrgh. if you want, go for it. but is it such a big deal? 😄 we are old and tired people 👵

Reply:
Yeah you're right. We all have better things to do than refactoring interfaces :smiley: